python/dask_cudf/dask_cudf/tests/test_groupby.py

```diff
@@ -610,7 +610,8 @@ def test_groupby_agg_params(npartitions, split_every, split_out, as_index):
         if as_index:
             # Groupby columns became the index.
             # Sorting the index should not change anything.
-            dd.assert_eq(gf.index, gf.sort_index().index)
+            with dask.config.set({"dataframe.convert-string": False}):
+                dd.assert_eq(gf.index, gf.sort_index().index)
```

Comment:
I think we just need to convert to frame for the string conversion to be consistent.
```suggestion
            dd.assert_eq(gf.index.to_frame(), gf.sort_index().index.to_frame())
```

Reply:
Made this change 👍